pelican/plugins/seo/settings.py

```diff
@@ -3,7 +3,7 @@
 """
 
 SEO_REPORT = True
-SEO_ENHANCER = True
+SEO_ENHANCER = False
```

Comment:
As said in the PR message, I propose that the default settings of the tool should not touch the generated pages to avoid surprises for the users.
The main comment in b37850b may be abusive, as it is not an error correction, but more the application of a general rule.
I think that most people want to have a report on their work first before they consent to explicitly alter it with a dedicated setting.

 I suggest the first of "default settings read-only vs write without explicit permission".

Reply:
Ok, I agree, let's go for read-only mode.